great_expectations/dataset/pandas_dataset.py

```diff
@@ -713,26 +708,27 @@ def matches_json_schema(val):
     @DocInherit
     @MetaPandasDataSet.column_aggregate_expectation
     def expect_column_mean_to_be_between(self, column, min_value=None, max_value=None,
-                                         output_format=None, include_config=False, catch_exceptions=None, meta=None):
+                                         result_format=None, include_config=False, catch_exceptions=None, meta=None):
 
         if min_value is None and max_value is None:
             raise ValueError("min_value and max_value cannot both be None")
 
         column_mean = column.mean()
 
         return {
-            "success": (
+            'success': (
```

Comment:
Are you using a linter? If so, which one? I need to add linting to my python dev loop.

Reply:
We're still inconsistent on this throughout, but I did change some instances where I encountered them inline. I preferred single ticks so we could embed expected json strings more naturally without escaping...